src/main/java/fr/xephi/authme/security/HashAlgorithm.java

```diff
@@ -31,6 +31,7 @@
     WORDPRESS(fr.xephi.authme.security.crypts.Wordpress.class),
     XAUTH(fr.xephi.authme.security.crypts.XAuth.class),
     XFBCRYPT(fr.xephi.authme.security.crypts.XfBCrypt.class),
+    CMW(fr.xephi.authme.security.crypts.Md5InSha1.class),
     CUSTOM(null),
```

Comment:
This enum is alphabetically ordered, besides the last `CUSTOM` entry.

Reply:
I commit the changes